### PR TITLE
DEVICE-885: More observability metrics for LTE

### PR DESCRIPTION
**Pull request by dafcioo**
Implements DEVICE-885
Requires:
- https://github.com/HaydenAI/device-scripts/pull/177
- https://github.com/HaydenAI/containers/pull/49

### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/outputs/proto/proto.pb.go [hunk 1]
```diff
@@ -9398,15 +9438,15 @@ var file_proto_proto_rawDesc = []byte{
 	0x6f, 0x73, 0x74, 0x12, 0x16, 0x0a, 0x06, 0x73, 0x6f, 0x75, 0x72, 0x63, 0x65, 0x18, 0x02, 0x20,
 	0x01, 0x28, 0x09, 0x52, 0x06, 0x73, 0x6f, 0x75, 0x72, 0x63, 0x65, 0x12, 0x12, 0x0a, 0x04, 0x70,
 	0x61, 0x74, 0x68, 0x18, 0x03, 0x20, 0x01, 0x28, 0x09, 0x52, 0x04, 0x70, 0x61, 0x74, 0x68, 0x22,
-	0xc6, 0x04, 0x0a, 0x03, 0x4c, 0x54, 0x45, 0x12, 0x29, 0x0a, 0x06, 0x66, 0x69, 0x65, 0x6c, 0x64,
```
[the remark this answers is not in the document]
Comment (dafcioo) [reply plugins/outputs/proto/proto.pb.go:9401]: I followed the instruction at the top of this file:
// Code generated by protoc-gen-go. DO NOT EDIT.

Thus, I installed the same version of protoc tool as used for generation of the previous version of the file and re-generated it by the command:
protoc --go_out=paths=source_relative:. proto.proto

[the remark this answers is not in the document]
Comment (thien-nguyen-fw) [reply plugins/outputs/proto/proto.pb.go:9401]: @TPiowczykHay How is this binary structure being used?

[the remark this answers is not in the document]
Comment (TPiowczykHay) [reply plugins/outputs/proto/proto.pb.go:9401]: This is autogenerated by protoc for protobuf serialization, nothing to care about on high level.
This is used in protobuf library to serialize the proto message and then we send it to influx-proto-proxy API service. Protobuf is backward compatible, so we don't need to have it updated on API service to handle messages.

If we want to have these new metrics saved in influx db (what API is responsible for) we will need to update the .proto on `hayden-cloud`.

[the remark this answers is not in the document]
Comment (dafcioo) [reply plugins/outputs/proto/proto.pb.go:9401]: Here is a ticket to update .proto file in hayden-cloud: CLOUD-3163

**Review by k-monika — APPROVED** [review]
Looks fine by me.

**Pull request merged**